spec/javascripts/jasmine_spec.rb

```diff
@@ -4,6 +4,10 @@
 # Run the jasmine tests by running the karma javascript test framework
 # The spec will fail if any jasmine tests fails.
 RSpec.describe "Jasmine" do
+  before do
+    `cd $RAILS_ROOT && bundle exec rake assets:precompile` if ENV['RAILS_ROOT']
```

Comment:
this is really expensive to do on every spec run... is there anything we can do to mitigate that cost?

Reply:
I can't find a way to make the JS tests work without compiling. This used to occur during the image build, so it's not a new time sink.